doc/source/index.rst

```diff
@@ -85,8 +85,165 @@ should have.
        cluster_arn="arn:aws:ecs:<region>:<acctid>:cluster/<gpuclustername>",
        worker_gpu=1)
 
+Azure
+-----
+
+In order to start using ``dask_cloudprovider.AzureMLCluster`` you need, at a minimum,
+an Azure subscription, an AzureML workspace, and a quota to create your compute target.
```

Comment:
Could you link `Azure subscription`, `AzureML workspace` and `quota` to some reference documentation on the Azure site?